lib/driver.js

```diff
@@ -412,10 +417,7 @@ class XCUITestDriver extends BaseDriver {
       await this.wda.uninstall();
       this.logEvent('wdaUninstalled');
     } else if (!util.hasValue(this.wda.webDriverAgentUrl) && (await this.wda.isRunning())) {
-      log.info(`Will reuse previously cached WDA instance at '${this.wda.url.href}'. ` +
```

Comment:
`await this.wda.isRunning()` is for old WDA. After https://github.com/appium/WebDriverAgent/pull/95 WDA, `await this.wda.isRunning()` is redundant method since we call `await this.getRunningBundleId()` in `uninstallWDAIfRunningBundleIdIsDifferent`. The cost is it increases the number of calling http request, `/status`, to twice.

Reply:
It would be more effective to just call getStatus and retrieve the resulting dict, so then we can analyze it content there directly of check if it is null without additional calls

Reply:
Yeah, right. Just came up with it. Let me update.